src/sql/azdata.proposed.d.ts

```diff
@@ -226,14 +234,11 @@ declare module 'azdata' {
 		 * A unique identifier for this item
 		 */
 		id: string;
+
 		/**
-		 * The optional icon to display for this item
-		 */
-		iconPath?: string;
-		/**
-		 * The other properties that will be displayed in the grid
+		 * The other properties that will be displayed in the grid columns
 		 */
-		[key: string]: string | DataGridHyperlinkInfo;
+		fieldValues: { [key: string]: string | DataGridHyperlinkInfo }
```

Comment:
Sorry about other things getting here - was making some other changes as I went along and it'd be too much of a pain to try and separate them out. It's usually pretty simple changes though.